includes/class-wc-gateway-dintero-hp.php

```diff
@@ -1361,11 +1358,17 @@ public function callback( $return_page = false ) {
 			if ( ! empty( $order ) && $order instanceof WC_Order ) {
 				$amount = absint( strval( floatval( $order->get_total() ) * 100 ) );
 				if ( array_key_exists( 'status', $transaction ) &&
-					 array_key_exists( 'amount', $transaction ) &&
-					 $transaction['amount'] === $amount ) {
-
-					if ( 'AUTHORIZED' === $transaction['status'] ) {
-
+					 array_key_exists( 'amount', $transaction )) {
+
+					if ($transaction['amount'] < $amount - 1 ) {
+						$hold_reason = sprintf(
+							'Order was authorized with a different amount, so manual handling is required. When captured in Dintero Backoffice, the order can be marked as completed. Transaction amount: %s. Order amount: %s. Transaction ID: %s. ',
```

Comment:
is it possible to add line breaks to the note, I think the formatting could be better

Reply:
yes!